types/stylelint/index.d.ts

```diff
@@ -192,6 +193,7 @@ declare namespace stylelint {
 	 * A rule context.
 	 */
 	type RuleContext = {
+		configurationComment?: string | undefined;
```

Comment:
[suggest] How about adding `configurationComment` to the developer guide?

https://github.com/stylelint/stylelint/blob/fcff497a3ea121a2721587f93f7b95d1eba3f6c9/docs/developer-guide/rules.md?plain=1#L198-L201

Reply:
Updated